test/Kestrel.Transport.FunctionalTests/RequestTests.cs

```diff
@@ -906,5 +900,38 @@ private async Task TestRemoteIPAddress(string registerAddress, string requestAdd
                 Assert.NotEmpty(facts["RemotePort"].Value<string>());
             }
         }
+
+        // THIS IS NOT GENERAL PURPOSE. If the initial characters could repeat, this is broken. However, since we're
+        // looking for /bytesWritten: \d+/ and the initial "b" cannot occur elsewhere in the pattern, this works.
+        private static async Task AssertStreamContains(Stream stream, string expectedSubstring)
```

Comment:
this can be consolidated in a shared source.

Reply:
I thought about putting this in StreamExtensions, but I really don't want this to be reused. At least not until it handles repeated character sequences.